api/src/test/java/org/openmrs/api/OrderServiceTest.java

```diff
@@ -3794,12 +3800,139 @@ public void saveOrderGroup_shouldFailValidationIfAnyOrdersFailValidation() {
 		Exception expectedGroupValidationError = null;
 		try {
 			Context.getOrderService().saveOrderGroup(orderGroup);
-		}
-		catch (Exception e) {
+		} catch (Exception e) {
 			expectedGroupValidationError = e;
 		}
 
 		assertNotNull(expectedGroupValidationError, "Validation should cause order group to fail to save");
 		assertEquals(expectedValidationError.getMessage(), expectedGroupValidationError.getMessage());
 	}
+	
+	@Test
+	public void getOrderGroupAttributeTypes_shouldReturnAllOrderGroupAttributeTypes(){
+		executeDataSet(ORDER_GROUP_ATTRIBUTES);
+		List<OrderGroupAttributeType>orderGroupAttributeTypes=orderService.getOrderGroupAttributeTypes();
+		assertEquals(4,orderGroupAttributeTypes.size());
+	}
+	
+	@Test
+	public void getOrderGroupAttributeType_shouldReturnNullIfNonExistingIdIsProvided(){
+		executeDataSet(ORDER_GROUP_ATTRIBUTES);
+		assertNull(Context.getOrderService().getOrderGroupAttributeTypeById(10));
+	}
+
+	@Test 
+	public void getOrderGroupAttributeType_shouldReturnOrderGroupAttributeTypeGivenId(){
+		executeDataSet(ORDER_GROUP_ATTRIBUTES);
+		final Integer ID = 2;
+		final String UUID2="9cf1bbe6-d18e-11ea-87d0-0242ac130003";
+		OrderGroupAttributeType orderGroupAttributeType = orderService.getOrderGroupAttributeTypeById(ID);
+		assertEquals(orderService.getOrderGroupAttributeTypeByUuid(UUID2),orderGroupAttributeType);
```

Comment:
Can you take a look at existing tests in core that test for returning an item by uuid?